class.js

```diff
@@ -143,19 +159,14 @@ class Pokemon {
   }
 
   evolve(){
-    //Basic Pokemon: Pikachu, Charmander, Squirtle, Bulbasaur, Pidgey, Caterpie, Ekans, Oddish, Doduo, Vulpix, Jigglypuff, Zubat
-    //2nd level Pokemon: Raichu, Charmeleon, Wartortle, Ivysaur, Pidgeotto, Metapod, Arbok, Gloom, Dodrio, Ninetales, Wigglytuff, Golbat
-    //3rd level: Charizard, Blastoise, Venusaur, Pigeot, Butterfree, Vileplume, Crobat
-    //2nd level unevolvable: Raichu, Arbok, Dodrio, Ninetales, Wigglytuff
-    
-    //Actually when it's already 3rd level it shouldn't try to evolve - so we just have to check for the other 5. EZ
-    if (this.name == "Raichu" || this.name == "Arbok" || this.name == "Dodrio" || this.name == "Ninetales" || this.name == "Wigglytuff" || this.name == "Clefable"){
+    if (this.name == "Raichu" || this.name == "Arbok" || this.name == "Dodrio" || this.name == "Ninetales" || this.name == "Wigglytuff" || this.name == "Clefable" || this.name == "Lapras" || this.name == "Raticate" || this.name == "Golduck"){
```

Comment:
I'd probably try to put these names in a list or dictionary rather than having this big logical expression. Grab me if you don't know what that means and we can go over it.